ui/app/components/secure-variable-form.js

```diff
@@ -201,7 +201,7 @@ export default class SecureVariableFormComponent extends Component {
   }
 
   get stringifiedItems() {
-    return JSON.stringify(this.args.model.items, null, 2);
+    return stringifyObject([this.args.model.items]);
```

Comment:
`nit`:  We could just call this directly in the template and avoid adding another variable to the template.